gdk4/src/auto/snapshot.rs

```diff
@@ -13,6 +13,10 @@ glib::wrapper! {
     }
 }
 
+impl Snapshot {
```

Comment:
This is not great, we ideally want a config option to disable generating the const

Reply:
Why do you want to disable it? It would be as necessary now as for other non-final types